forge/tests/it/repros.rs

```diff
@@ -179,6 +179,27 @@ fn test_issue_3055() {
         }
     }
 }
+// <https://github.com/foundry-rs/foundry/issues/3192>
+#[test]
+fn test_issue_3192() {
+    let mut runner = runner();
+    let suite_result =
+        runner.test(&Filter::new(".*", ".*", ".*repros/Issue3192"), None, TEST_OPTS).unwrap();
+    assert!(!suite_result.is_empty());
```

Comment:
We have a lot of this boilerplate at this point, should we convert this into a function `test_issue(num: u64, expected: bool)`?

Reply:
yeh we definitely want this, will write additional helpers to make this nicer